packages/paste-core/components/input-box/src/InputChevronWrapper.tsx

```diff
@@ -1,18 +1,34 @@
 import * as React from 'react';
+import * as PropTypes from 'prop-types';
 import {Box} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
 
-export const InputChevronWrapper = React.forwardRef<HTMLDivElement, {children: React.ReactNode}>(({children}, ref) => (
-  <Box
-    alignItems="center"
-    display="inline-flex"
-    position="absolute"
-    pointerEvents="none"
-    right="space30"
-    top="50%"
-    transform="translateY(-50%)"
-    zIndex="zIndex10"
-    ref={ref}
-  >
-    {children}
-  </Box>
-));
+export interface InputChevronWrapperProps extends Pick<BoxProps, 'element'> {
+  children: NonNullable<React.ReactNode>;
+}
+
+export const InputChevronWrapper = React.forwardRef<HTMLDivElement, InputChevronWrapperProps>(
+  ({children, element}, ref) => (
+    <Box
+      alignItems="center"
+      display="inline-flex"
+      position="absolute"
+      pointerEvents="none"
+      right="space30"
+      top="50%"
+      transform="translateY(-50%)"
+      zIndex="zIndex10"
+      element={`${element}_CHEVRON_WRAPPER`}
```

Comment:
We might need to be a little careful here. `element` from Box is optional, and I know we only use these as a compositional element to Input, but I still think we should either:

- Set a default element name in the arguments
- Make `element` required so that it's never NOT passed (preferred)